### ISSUE TITLE
Fix overlay

### ISSUE THREAD

**Issue by finestructure** [finestructure/PreviewProviderTools#4]
Recent introduction of the background overlay (`ZStack`) has broken nesting, I believe:

```
// Playground generated with 🏟 Arena (https://github.com/finestructure/arena)
// ℹ️ If running the playground fails with an error "no such module ..."
//    go to Product -> Build to re-trigger building the SPM package.
// ℹ️ Please restart Xcode if autocomplete is not working.

import Gala
import PlaygroundSupport
import SwiftUI

struct ContentView: View {
    var body: some View {
        VStack {
            Text("🎭 Gala").font(.title).padding()
            Text("🎭 Foo").font(.title).padding()
        }
    }
}

struct Preview: PreviewProvider {
    static var previews: some View {
        NightAndDay {
            Layouts([.sizeThatFits, .fixed(width: 200, height: 200)]) {
                ContentView()
            }
        }
    }
}

PlaygroundPage.current.setLiveView(Preview.previews)
```

<img width="176" alt="Screenshot 2020-04-07 at 07 55 08" src="https://user-images.githubusercontent.com/65520/78634824-1b3ad980-78a5-11ea-874b-234358e94b87.png">

This should be showing four tiles and I believe it isn't, because they're overlaid.

**Comment by finestructure** [comment]
This might be fixed by #7 as well

**Comment by finestructure** [comment]
Indeed it is fixed in 0.3.2:

<img width="1072" alt="Screenshot 2020-05-23 at 10 04 56" src="https://user-images.githubusercontent.com/65520/82725380-df38c800-9cdc-11ea-9a8a-5fe203a31b38.png">
